web/models/user.ex

```diff
@@ -35,4 +35,14 @@ defmodule RemoteRetro.User do
     |> unique_constraint(:email)
     |> validate_format(:email, ~r/@/)
   end
+
```

Comment:
would also be nice to get some tests around this, as it's part of `User`'s public interface